release/release.go

```diff
@@ -187,6 +187,11 @@ var OnClassic bool
 // OnCoreDesktop states whether the process is running inside a Core Desktop image.
 var OnCoreDesktop bool
 
+// OnTouch states whether the process is running inside a Touch image
+// with a classic (but mostly read-only) filesystem layout.
+// It is a narrowed down variant of OnClassic
+var OnTouch bool
```

Comment:
I think I'd split the part specific to Touch into a separate PR. There may be questions about how we want to approach this specific variant that are unrelated to supporting libhybris

Reply:
Tbf I've waited pretty long on the previous PR that has been sitting for a while, which used to be the libhybris-only PR. But if more people want to have it split up, then sure.

Reply:
I would split it. The s-c bits should be fairly easy to land, but folks may be asking questions about this one and there's no point in holding back the rest of the PR.

Reply:
Done.